examples/CSharp/OpenGL Tutorials/Tutorial 2.2 - Camera/Program.cs

```diff
@@ -184,6 +185,11 @@ private static unsafe void OnRender(double deltaTime)
             Gl.DrawArrays(PrimitiveType.Triangles, 0, 36);
         }
 
+        private static void OnFramebufferResize(Vector2D<int> newSize)
```

Comment:
Don't these tutorials have size-based uniforms to update as well? (search for usage of the Width and Height constants)

Reply:
Yep, fixing it.